src/GUI/EFCorePowerTools/Extensions/ProjectExtensions.cs

```diff
@@ -66,17 +66,20 @@ public static List<string> GetConfigFiles(this Project project)
                 return result;
             }
 
-            var file = Directory.GetFiles(projectPath, "efpt.config.json");
-            result.AddRange(file);
+            //var file = Directory.GetFiles(projectPath, "efpt.config.json");
```

Comment:
I think you need to leave this in, if there are any special config files and the standard one, the standard one will be missed if you do not include these 2 lines

Reply:
no need.
 "efpt.*config.json" will get 
efpt.config.json
efpt.sample.config.json
efpt.sample2config.json

alway get the standart one

Reply:
Yes, but this method must return at least on  entry, even if no file exists! Look at the logic in the caller.

Reply:
and will, the rest of the code ensures that

            if (result.Count() == 0)
            {
                result.Add(Path.Combine(projectPath, "efpt.config.json"));
            }

Reply:
OK! 😅